docs/cloud/azureai/use-flow-in-azure-ml-pipeline.md

```diff
@@ -133,8 +196,50 @@ Above is part of the pipeline job python code, see here for [full example](https
+|                  | environment_variables.\<environment-variable-name\> | from environment variables defined in `run.yaml` | string                  | default value will be the current value defined in `run.yaml`; to override environment variables during flow run, e.g. OPEN_AI_API_KEY. Note that you can refer to workspace connections with expressions like `{my_connection.api_key}`. |
+| output ports     | flow_outputs                                        | fixed                                            | uri_folder              | an uri_folder with 1 or more jsonl files containing outputs of the flow runs |
+|                  | debug_info                                          | fixed                                            | uri_folder              | an uri_folder containing debug information of the flow run, e.g., run logs |
+| run settings     | instance_count                                      | fixed (all run setting ports are fixed)          | integer                 | The number of nodes to use for the job. Default value is 1.  |
+|                  | max_concurrency_per_instance                        | the same as above                                | integer                 | The number of processors on each node.                       |
+|                  | mini_batch_error_threshold                          | the same as above                                | integer                 | Define the number of failed mini batches that could be ignored in this parallel job. If the count of failed mini-batch is higher than this threshold, the parallel job will be marked as failed.<br/><br/>Mini-batch is marked as failed if:<br/>- the count of return from run() is less than mini-batch input count.<br/>- catch exceptions in custom run() code.<br/><br/>"-1" is the default number, which means to ignore all failed mini-batch during parallel job. |
+|                  | retry_settings.max_retries                          | the same as above                                | integer                 | Define the number of retries when mini-batch is failed or timeout. If all retries are failed, the mini-batch will be marked as failed to be counted by `mini_batch_error_threshold` calculation. |
+|                  | retry_settings.timeout                              | the same as above                                | integer                 | Define the timeout in seconds for executing custom run() function. If the execution time is higher than this threshold, the mini-batch will be aborted, and marked as a failed mini-batch to trigger retry. |
+|                  | logging_level                                       | the same as above                                | INFO, WARNING, or DEBUG | Define which level of logs will be dumped to user log files. |
```

Comment:
maybe add some example (YAML & SDK), with some typical runsetting

Reply:
added in previous section